GVFS/GVFS.FunctionalTests/Settings.cs

```diff
@@ -35,7 +35,7 @@ public static void Initialize()
                 CurrentDirectory = Path.GetFullPath(Path.GetDirectoryName(Environment.GetCommandLineArgs()[0]));
 
                 RepoToClone = @"https://gvfs.visualstudio.com/ci/_git/ForTests";
-                Commitish = @"FunctionalTests/20180214";
+                Commitish = @"FunctionalTests/20190827";

```

Comment:
For the benefit of the uninitiated (me), what does this do?

Reply:
Great question ;-), I had to 2 new files to the 20190827 branch (wrapper.sh / test.py) to perform the test.  Generally if we add new files we create a new branch.

@kewillford to please validate this change.